### PR TITLE
Added nukeops victory state for failing nuke but hijacking the shuttle

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Added a win state for failing to nuke the ship, but managing to hijack the emergency shuttle, with disk securing or not as small alternatives.

## Why It's Good For The Game

I almost saw this happen earlier today and it amused me enough to make a PR on the off-chance it actually happens sometime ingame, like the in-code victory state for the crew managing to nuke the syndie base.

## Changelog

:cl:
add: Added a nukeops victory state for failing to nuke the station, but somehow hijacking the shuttle.
/:cl:

### CODE REVIEW

**Comment by VladinXXV** [comment]
I think this should _require_ securing the disk- or at the very least, disky should be on shuttle/off station. If the disk is still on the station, you still have a job to do, operative! I'm concerned about this encouraging operatives to abandon their primary objective in favor of "lol 5 viscerator grenades in shuttle cry about it" or similar, y'know. They are nuclear operatives, not hijack operatives.

**Comment by carlarctg** [comment]
> 

overthinking

File: code/modules/antagonists/nukeop/nukeop.dm [hunk 1]
```diff
@@ -429,6 +437,12 @@
 		if(NUKE_RESULT_WRONG_STATION_DEAD)
 			parts += "<span class='redtext big'>[syndicate_name] operatives have earned Darwin Award!</span>"
 			parts += "<B>[syndicate_name] operatives blew up something that wasn't [station_name()] and got caught in the explosion.</B> Next time, don't do that!"
+		if(NUKE_RESULT_HIJACK_DISK)
+			parts += "<span class='greentext big'>Syndicate... Minor Victory?</span>"
```
Comment (san7890) [inline code/modules/antagonists/nukeop/nukeop.dm:441]:
```suggestion
			parts += "<span class='greentext big'>Syndicate Insignificant Victory!</span>"
```

**Review by san7890 — CHANGES_REQUESTED** [review]
I think canonically the Syndicate being able to acquire a Nanotrasen shuttle potentially full of crew that were likely armed to the teeth with experimental technology would still work in their interest in some way.

I get the humorous angle with having the greentext announcer be unsure of what to actually call the outcome, but I think it looks much better if it's certain of what happened and reacts appropriately.

**Pull request merged**